fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -913,7 +914,7 @@ public Bundle doSearch(String type, String compartment, String compartmentId,
             // Check to see if it's supported, else, throw a bad request.
             // If this is removed, it'll result in nullpointer when processing the request
             if (!ModelSupport.isResourceType(type)) {
-                throw buildUnsupportedResourceTypeException(type, IssueType.NOT_SUPPORTED);
+                throw buildUnsupportedResourceTypeException(type);
             }
```

Comment:
there is a high amount of duplication, just a _small suggestion_ to push it down into the method `buildUnsupportedResourceTypeException` and make it a void method.